adagrams/game.py

```diff
@@ -1,13 +1,182 @@
 from random import randint
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+def convert_letter_pool_to_list(letter_pool):
+    letters_list =[]
+    for letter, count in letter_pool.items():
+        for i in letter * count:
+            letters_list.append(i)
+    return letters_list
```

Comment:
Love that you used a helper function here! Makes your code less susceptible hard to find bugs! A couple of things here:
- You'll often see a space between variable declaration and other parts of code. In a more generalized manner, you will find that developers will put spaces between blocks of code that serve different purposes. A loop that iterates to find certain elements and a loop a that creates a frequency map would have a space between to visually signal that they have different purposes.
- We could combine the functionality of inner loop into one line by utilizing the `.extend` method available to us via lists. Read more on it[ here](https://docs.python.org/3/tutorial/datastructures.html#more-on-lists)
```suggestion
    letters_list =[]
    
    for letter, count in letter_pool.items():
            letters_list.append(list(letter) * count)
    return letters_list
```

Reply:
Thanks for showing other ways! I will read more about `.extend` method